technico-v16/app/components/LogOutButton.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { useAuth } from "../components/AuthContext";
+
```

Comment:
- Preferably the tokern should be removed
- Good use of fireReload to trigger auth state update and redirect afterwards